<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab6Class</class>
 <widget class="QWidget" name="lab6Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab6</string>
  </property>
  <widget class="QPushButton" name="QuitBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QWidget" name="MainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 1px; border-color: blue;</string>
       </property>
       <property name="text">
        <string>Enter your number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputForm">
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 1px; border-color: green;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="head1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-width: 1px; border-color: blue;</string>
    </property>
    <property name="text">
     <string>Choose input numerical system</string>
    </property>
   </widget>
   <widget class="QLabel" name="head2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-width: 1px; border-color: blue;</string>
    </property>
    <property name="text">
     <string>Choose output numerical system</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="output">
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 2px; border-color: green;</string>
       </property>
       <property name="text">
        <string>Waiting for input...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="headOut">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="inSys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="currentText">
     <string>10</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="outSys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="currentText">
     <string>2</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lab6.qrc"/>
 </resources>
 <connections/>
</ui>
